samples/nrf9160/aws_fota/README.rst

```diff
@@ -74,24 +74,16 @@ Updating the certificates
 =========================
 
 The certificates that you created or added for your thing in AWS IoT must be stored on your kit so that it can connect to AWS IoT.
-There are two different ways of doing this:
 
-Add the certificates to the sample code:
-   If you add the certificates to the sample code, the sample will store them on your kit automatically.
+Use `LTE Link Monitor`_ to write the certificates to the kit.
+This application, which is part of `nRF Connect for Desktop`_, provides a certificate manager that you can use to store the certificates on your kit:
 
    .. warning_start
 
    .. warning::
-      * The sample will overwrite the certificates stored with the configured :option:`security tag <CLOUD_CERT_SEC_TAG>`.
-      * You should provision the certificates only once and then update the sample configuration to use the existing certificates.
-        When provisioning the certificates, they are stored in the application binary and visible in the modem trace information, which is a security risk.
-
+      * The you could overwrite the certificates stored on the device with LTE Link monitor so eb sure to use the correct security tag number.
```

Comment:
What?
Maybe:
```suggestion
      The LTE Link Monitor will overwrite the certificates stored on the device with the given security tag number.
      Therefore, make sure that you are using the correct security tag.
```
Also, only one warning left, so no bullets.

Reply:
Seems like i pushed something old on top of some changes sorry. Agreed (What?)